src/qt/transactionfilterproxy.h

```diff
@@ -35,7 +35,7 @@ class TransactionFilterProxy : public QSortFilterProxyModel
     };
 
     void setDateRange(const QDateTime &from, const QDateTime &to);
-    void setAddressPrefix(const QString &addrPrefix);
+    void setSearchString(const QString &);
```

Comment:
`const QString&`.

Reply:
Nit: I prefer having the variable names in the header (all others in this header do too). Unsure if there's a general preference though